virttest/nbd.py

```diff
@@ -106,8 +119,12 @@ def export_image(qemu_nbd, filename, local_image, params):
                 tls_creds=params['nbd_server_tls_creds']
             )
 
-    if params.get('nbd_export_bitmap'):
-        cmd_dict['bitmap'] = '-B %s' % params['nbd_export_bitmap']
+    # multiple bitmaps can be exported
+    bitmap_setting = ''
+    for bm in params.objects('nbd_export_bitmap'):
+        bitmap_setting += ' -B %s' % bm
+    if bitmap_setting:
+        cmd_dict['bitmap'] = bitmap_setting.strip()
```

Comment:
```suggestion
    if params.get('nbd_export_bitmaps'):
        cmd_dict['bitmap'] = "".join([" -B %s" % _ for _ in params['nbd_export_bitmap'].split()])
```
Your codes look good for me, just I prefer using `nbd_export_bitmaps` to clarify your intention to 
tell the user who could assign multiple bitmaps by the name of `nbd_export_bitmaps`, like images, vms, nics, etc.
Thanks.

Reply:
You are right, yongxue, at the very beginning, I still use the original param name, which can avoid some case-level code change, in the end, I forgot to make the change, I will update it later. Thanks much.